front-end/src/components/blog/MarkdownViewer.vue

```diff
@@ -328,9 +333,16 @@ if (!props.markdown) {
 	onMounted(async () => {
 		await getArticleById(_id.value);
 		srcMarkdown.value = oneItems.value.content;
-		date.value = formatDateField(oneItems.value.date);
+		date.value = oneItems.value.date;
 		title.value = oneItems.value.title;
 		description.value = oneItems.value.descriptions;
+    let tempUser = await materialStore.getUserById(oneItems.value.owner)
+
+    // format for captital letters
+    user.value = 
+      tempUser.profile.firstName.charAt(0).toUpperCase() + tempUser.profile.firstName.slice(1)
```

Comment:
There's really no need to do that in js.
In css you have text-transform: capitalize;